src/status_im/activity_center/core.cljs

```diff
@@ -86,18 +86,23 @@
   (and (some? cursor)
        (not= cursor start-or-end-cursor)))
 
-(defn- filter-status->rpc-method
-  [filter-status]
-  (if (= filter-status :read)
-    "wakuext_readActivityCenterNotifications"
-    "wakuext_unreadActivityCenterNotifications"))
+(def ^:const status-read 1)
```

Comment:
Do you have a doc reference as to why we should tell the compiler these things are constants? Does it bring any meaningful performance improvement?

Reply:
> Does it bring any meaningful performance improvement?

it doesn't